docs/application/native/guides/app-management/widget-service.md

```diff
@@ -0,0 +1,125 @@
+# Widget Service
+
+
+The Widget Service API provides functionality to get the information of installed widgets.
+
+The main features of the Widget Service API include:
+
+- Retrieving information of widgets
+
+  You can [retrieve the information](#retrieve) of an installed widget application.
+
+- Communicating with running widget instances
+
+  You can [communicate](#communicate) with running widget instances.
+
+
+
+> [!NOTE]
+>
+> This API will works only if the widget and application are packaged together.
```

Comment:
```suggestion 
> [!NOTE]
>  Widget Service API works only if the widget and application are packaged together.
```
- There is no space between note & text 
- I have slightly modified sentence for correction in grammar.